PaddleHQ/transaction-search
# Make state parameter camel case

### Checklist

### What has been done:
Similar to the previous PR: the `state` parameter has as its value a constant. To make it easier to parse that constant, the case is changed to lowerCamelCase.

### How to test:
1. Find a partially refunded transaction
1. Make a request with parameters that would result in your transaction
1. Confirm that the state is `partiallyRefunded`
1. Confirm with other states too.
